### ISSUE TITLE
Swiftwolf rending gale not allowing me to fly 

### ISSUE THREAD

**Issue by robohavier32** [sinkillerj/ProjectE#2287]
**Tickets that do not conform to this template will be closed without comment**

ProjectE-1.18.2-PE1.0.1.jar

Forge 40.2.9

There is no crash.

Steps to reproduce:

1. Install the version I mentioned above on forge 1.18.2 modded.

What I expected to happen: I expected to be able to fly with the swift wolf-rending gale

What happened instead: It didn't allow me to fly even though I had fuel. It did let me use the other abilities though.

**Comment by robohavier32** [comment]
It turns blue in creative but not in survival.

**Comment by DINGLELOID** [comment]
i have the exact same issue ive tried every fuel and nothing works

**Comment by SDPhantom** [comment]
To my knowledge, nothing in the mod consumes "fuel" to operate. Everything that takes EMC to run requires a Klein Star that has a sufficient amount of EMC stored in it. The easiest way to charge one is to put it in one of the slots on the left side of the transmutation table/tablet GUI. This will transfer EMC stored in the table/tablet into the Klein Star. To transfer it back, you can either put the Klein Star in the burn slot, consuming it, or put it in the center filter slot on the right side of the GUI, which will drain it.

**Comment by PlasticGlove** [comment]
same exact problem here

**Comment by PlasticGlove** [comment]
when i swap from creative to survival (via LAN) it allows me to fly in survival like it normally functions UNTIL i touch the ground, then it wont let me fly anymore. The other abelites work like the lightning cast and bubble shield, but not fly.

**Comment by robohavier32** [comment]
That part. I don’t understand why the moderators can’t understand that

**Comment by robohavier32** [comment]
> when i swap from creative to survival (via LAN) it allows me to fly in survival like it normally functions UNTIL i touch the ground, then it wont let me fly anymore. The other abelites work like the lightning cast and bubble shield, but not fly.

Bubble shield?

**Comment by robohavier2** [comment]
> when i swap from creative to survival (via LAN) it allows me to fly in survival like it normally functions UNTIL i touch the ground, then it wont let me fly anymore. The other abelites work like the lightning cast and bubble shield, but not fly.

Issue might be with the mod magic rings if you have it or magical jewelry check mods like those since the moderators still haven't closed this issue.

**Comment by darthvader45** [comment]
I've had the same problem, except with the hurricane/gem boots (which ofc use 2 of the rending gales to make).  I don't have magic rings tho.

**Comment by robohavier** [comment]
> I've had the same problem, except with the hurricane/gem boots (which ofc use 2 of the rending gales to make). I don't have magic rings tho.
It’s bugging me that so many people are having the same problem hopefully they add better compatibility I’m so irritated

**Comment by darthvader45** [comment]
Yeah, this needs to get fixed soon.  Can't stand having to glide from place to place (the ring weirdly works fine for me, and I'm using 1.1.0 of ProjectE.
